package.json

```diff
@@ -8,8 +8,8 @@
     "coverage-clear": "rm -Rf coverage",
     "coverage-check": "yarn coverage-generate && istanbul check-coverage --config .istanbul.yml",
     "coverage-check-unit": "yarn coverage-generate-unit && istanbul check-coverage --config .istanbul.unit.yml",
-    "coverage-generate": "yarn coverage-clear && yarn istanbul cover -- _mocha -- --recursive",
-    "coverage-generate-unit": "yarn coverage-clear && yarn istanbul cover -- _mocha -- --recursive test/unit",
+    "coverage-generate": "yarn coverage-clear && yarn istanbul -- cover _mocha -- --recursive",
```

Comment:
Not sure why this change is required as based on this https://github.com/gotwarlost/istanbul#the-cover-command the `old` way looks fine.

Reply:
My bad, this needs sorting.